### ISSUE TITLE
Cannot import pdf

### ISSUE THREAD

**Issue by tdse13** [paperless-ngx/paperless-ngx#1952]
### Description

Paperless does not accept pdf.

### Steps to reproduce

1. Import via GUI
2. Error message with name of pdf
3. Error Message: Not sure how to handle PDF image of this type
Or
1. Import via "consume"
2. Error message without file name
3. Error Message: Not sure how to handle PDF image of this type

### Webserver logs

```bash
[2022-11-09 11:51:42,534] [INFO] [paperless.management.consumer] Adding /usr/src/paperless/consume/535161001.pdf to the task queue.
[2022-11-09 11:51:42,550] [ERROR] [paperless.handlers] Creating PaperlessTask failed: connection already closed
11:51:42 [Q] INFO Enqueued 1
11:51:42 [Q] INFO Process-1:3687 processing [535161001.pdf]
11:51:42 [Q] INFO Process-1:3687 stopped doing work
11:51:43 [Q] ERROR Failed [535161001.pdf] - Not sure how to handle PDF image of this type : Traceback (most recent call last):
  File "/usr/src/paperless/src/src/django-q/django_q/cluster.py", line 454, in worker
    res = f(*task["args"], **task["kwargs"])
  File "/usr/src/paperless/src/documents/tasks.py", line 99, in consume_file
    pdf_filepath, separators = barcodes.scan_file_for_separating_barcodes(path)
  File "/usr/src/paperless/src/documents/barcodes.py", line 126, in scan_file_for_separating_barcodes
    pillow_img = pdfimage.as_pil_image()
  File "/usr/local/lib/python3.9/site-packages/pikepdf/models/image.py", line 742, in as_pil_image
    im = self._extract_transcoded()
  File "/usr/local/lib/python3.9/site-packages/pikepdf/models/image.py", line 606, in _extract_transcoded
    if self.mode in {'DeviceN', 'Separation'}:
  File "/usr/local/lib/python3.9/site-packages/pikepdf/models/image.py", line 301, in mode
    raise NotImplementedError(
NotImplementedError: Not sure how to handle PDF image of this type
11:51:46 [Q] INFO recycled worker Process-1:3687
11:51:46 [Q] INFO Process-1:3689 ready for work at 18466
```

### Browser logs

```bash
Not sure how to handle PDF image of this type : Traceback (most recent call last):
  File "/usr/src/paperless/src/src/django-q/django_q/cluster.py", line 454, in worker
    res = f(*task["args"], **task["kwargs"])
  File "/usr/src/paperless/src/documents/tasks.py", line 99, in consume_file
    pdf_filepath, separators = barcodes.scan_file_for_separating_barcodes(path)
  File "/usr/src/paperless/src/documents/barcodes.py", line 126, in scan_file_for_separating_barcodes
    pillow_img = pdfimage.as_pil_image()
  File "/usr/local/lib/python3.9/site-packages/pikepdf/models/image.py", line 742, in as_pil_image
    im = self._extract_transcoded()
  File "/usr/local/lib/python3.9/site-packages/pikepdf/models/image.py", line 606, in _extract_transcoded
    if self.mode in {'DeviceN', 'Separation'}:
  File "/usr/local/lib/python3.9/site-packages/pikepdf/models/image.py", line 301, in mode
    raise NotImplementedError(
NotImplementedError: Not sure how to handle PDF image of this type
```

### Paperless-ngx version

1.9.2

### Host OS

Synology DSM 7.1.1-42962

### Installation method

Docker - official image

**Comment by shamoon** [comment]
Its not very easy to look into this without a pdf. Most likely the PDF is malformed, you can try re-saving / re-generating it and trying again.

**Comment by tdse13** [comment]
The origin of the pdf is from a bank (WISE). Therefore, I cannot attach it. It was generated by Text 2.1.0 (by lowagie.com). However, I assume that  bank can produce valid pdfs.

**Comment by tredick** [comment]
Look like a duplicate to #1743 to me. 

I've had similar problems with some PDF (which could be opened in viewers without problems). My workaround was to print them again to the PDF printer of Windows 11. The resulting file could be consumed then.

That's not nice but since it seems bug #1743 which already has been fixed and only waits for the next release it probably won't be needed very long anymore.

**Comment by stumpylog** [comment]
I've gone ahead and made another change around barcodes.  Basically, we'll try very hard to look for barcodes, using the pikepdf method first, but if it can't handle the images, fallback to page by page image conversion.

**Comment by stumpylog** [comment]
Fixed via #1953
